### DISCUSSION TITLE
Generating full CSS

### DISCUSSION THREAD

**Discussion by srid** [windicss/windicss#391] [Q&A]
Is there a way to tell the WindiCSS generator to generate the full CSS for all Tailwind classes? Normally I could write HTML and use *every* tailwind class and every combination, but that's laborious. 

You may ask, where would I use this CSS? I would be using this in development mode (thus obviating having to shell out to windicss every time the page changes; it feels icky to spawn nodejs from Haskell), and use the windicss compiler only whe pushing to production.

Software in question that does this: https://note.ema.srid.ca/

**tldr;** Where can I get the equivalent of Tailwind CDN css (full dump) but for WindiCSS? (Yes, I can use Tailwind's CDN css itself, but that's missing windi-specific defaults like [these color variants](https://windicss.org/utilities/colors.html)).

**Comment by schelmo** [comment]
you can use https://github.com/antfu/windicss-runtime-dom
```html
<script src="https://unpkg.com/windicss-runtime-dom"></script>
```
which generates your classes on the fly.
generating the whole css file would be too big because there are too many variants
